packages/reports/src/Components/Form/CompositeCalendar/side-list.tsx

```diff
@@ -1,16 +1,28 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { toMoment } from '@deriv/shared';
 import ListItem from './list-item.jsx';
 
-const isActive = (from, to, flag) => {
+type TItem = {
+    value: string;
+    label: string;
+    onClick: () => void;
+    duration: number;
+};
+
+type TSideList = {
+    from: string;
+    items: Array<TItem>;
+    to: number;
+};
+
+const isActive = (from: string, to: number, flag: number) => {
```

Comment:
```suggestion
const isActive = (from: number, to: number, flag: number) => {
```

Reply:
I will change it 🙇🏼  back to the original